src/toil_scripts/rnaseq_cgl/rnaseq_cgl_pipeline.py

```diff
@@ -15,7 +15,7 @@
 import yaml
 from bd2k.util.files import mkdir_p
 from bd2k.util.processes import which
-from toil.job import Job
+from toil.job import Job, PromisedRequirement as PReq
```

Comment:
It looks like only line 79 is really affected by this, and it still is longer than allowed (116 < 100). For the sake of readability, I'll buy you two tacos if you use the original name …